src/auth/components/game-img/game-img.ts

```diff
@@ -0,0 +1,37 @@
+import Vue from 'vue';
+import Component from 'vue-class-component';
+import { State } from 'vuex-class';
+import AppGameThumbnailImg from '../../../_common/game/thumbnail-img/thumbnail-img.vue';
+import { Store } from '../../store/index';
+
+@Component({
+	name: 'GameCoverCredits',
+	components: {
+		AppGameThumbnailImg,
+	},
+})
+export default class GameCoverCredits extends Vue {
```

Comment:
Always name component classes with App as a prefix. Also, make sure the filename matches the class name. So the filename should be `game-cover-credits` and the class should be `AppGameCoverCredits`